daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala

```diff
@@ -25,7 +25,10 @@ object LoadDarFunction extends App {
         case (pkgId, pkgArchive) => Decode.readArchivePayloadAndVersion(pkgId, pkgArchive)._1
       }.toMap
 
-    val compiledPackages: CompiledPackages = PureCompiledPackages(packagesMap).right.get
+    val stacktracing: Compiler.StackTraceMode = Compiler.NoStackTrace
```

Comment:
This line gives _zero_ indication that it is overwriting a virtual method and not declaring a local binding. Scala really sucks in this regard. 😦

Reply:
Indeed I did not see that was overwriting a virtual method. 
In the future, I will ask you to use @S11001001 convention and mark all overwrite method/field (virtual or not) with the `override` keyword.

Reply:
Is this line overwriting a virtual method? I don't think so. It is just a local binding.
That's the intention anyway.

Reply:
It is indeed just a local binding. 😕

Reply:
My bad.